### PR TITLE
modify k8s executor jobname when remove

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
modify k8s executor jobname when remove

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=204366&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sImFzc2lnbmVlSURzIjpbIjEwMDEyMDUiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=467&type=BUG)

### CODE REVIEW

File: modules/pipeline/pipengine/actionexecutor/plugins/scheduler/executor/plugins/k8sjob/k8sjob.go [hunk 1]
```diff
@@ -197,12 +197,7 @@ func (k *K8sJob) Remove(ctx context.Context, task *spec.PipelineTask) (data inte
 		return nil, err
 	}
 
-	kubeJob, err := k.generateKubeJob(job)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to remove k8s job")
-	}
-
-	name := kubeJob.Name
+	name := strutil.Concat(task.Extra.Namespace, ".", task.Extra.UUID)
```
Comment (sfwn) [inline modules/pipeline/pipengine/actionexecutor/plugins/scheduler/executor/plugins/k8sjob/k8sjob.go:200]: why join with dot ?

File: modules/pipeline/pipengine/actionexecutor/plugins/scheduler/scheduler.go [hunk 2]
```diff
@@ -386,6 +386,7 @@ func (s *Sched) Cancel(ctx context.Context, action *spec.PipelineTask) (data int
 	}
 	if !shouldDispatch {
 		logrus.Infof("task executor %s execute cancel", taskExecutor.Name())
+		action.Extra.UUID = task_uuid.MakeJobID(action)
```
Comment (sfwn) [inline modules/pipeline/pipengine/actionexecutor/plugins/scheduler/scheduler.go:389]: How about other actionexecutors?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/pipengine/actionexecutor/plugins/scheduler/scheduler.go (hunk 2 above)
Comment (chengjoey) [reply modules/pipeline/pipengine/actionexecutor/plugins/scheduler/scheduler.go:389]: move makeJobID to framework may change task uuid in database

Comment (sfwn) [reply modules/pipeline/pipengine/actionexecutor/plugins/scheduler/scheduler.go:389]: put it in the comment.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**